ksy_schema.json

```diff
@@ -339,6 +339,7 @@
     "EnumValueSpec": {
       "$ref": "#/definitions/Identifier",
       "anyOf": [
+        { "type": "string" },
```

Comment:
The simple enum case was supposed to be handled by the `"$ref": "#/definitions/Identifier"` two lines above, but using `"$ref"` like that is not actually allowed by JSON Schema. I think the proper fix here would be to move the `"$ref": "#/definitions/Identifier"` from above into this object (in place of `"type": "string"`).